lesson_09/oop/oop_app/src/main/java/com/codedifferently/lesson9/chukwumaibezim/Patron.java

```diff
@@ -0,0 +1,31 @@
+package com.codedifferently.lesson9.chukwumaibezim;
+
+import java.util.HashSet;
+
+public class Patron {
+  private String name;
+  private HashSet<Book> checkedOutBooks;
+
+  public Patron(String name) {
+    this.name = name;
+    this.checkedOutBooks = new HashSet<>();
+  }
+
+  public String getName() {
+    return name;
+  }
+
+  public HashSet<Book> getCheckedOutBooks() {
+    return checkedOutBooks;
+  }
+
+  public void checkOutBook(Book book) {
```

Comment:
How would you prevent someone from using a `Patron` instance to check out a book without using the `Library`?

Reply:
Not seeing how this is resolved, did you make a change and forget to push?